README.md

```diff
@@ -79,7 +79,8 @@ No need to walk away to take a break, just sit comfortably, run Nafas and let th
 
 ## Usage	
 
-ℹ️ You can use `nafas`, `python -m nafas` or `NAFAS.exe` to run this program
+ℹ️ You can use `nafas`, `python -m nafas` or `NAFAS.exe` to run this 
```

Comment:
Did you accidentally remove **program**, or not?

Reply:
Yes, it was accidentally.